tooling/build/scripts/publishing/index.ts

```diff
@@ -0,0 +1,188 @@
+import { Client } from "pg";
+import * as fs from "fs";
+import * as path from "path";
+import * as dotenv from "dotenv";
+import { performance } from "perf_hooks";
+import {
+  GET_ALL_RESOURCES_WITH_FULL_PERMALINKS,
+  GET_NAVBAR,
+  GET_FOOTER,
+  GET_CONFIG,
+} from "./queries";
+
+dotenv.config();
+
+// Env vars
+const DB_USERNAME = process.env.DB_USERNAME;
+const DB_PASSWORD = process.env.DB_PASSWORD;
+const DB_HOST = process.env.DB_HOST;
+const DB_PORT = process.env.DB_PORT;
+const DB_NAME = process.env.DB_NAME;
+const SITE_ID = Number(process.env.SITE_ID);
+
+interface Resource {
+  id: number;
+  title: string;
+  permalink: string;
+  parentId: number | null;
+  type: string;
+  content?: any;
+  fullPermalink?: string;
+}
+
+// Wrapper function for debug logging
+function logDebug(message: string, ...optionalParams: any[]) {
+  if (process.env.DEBUG === "true") {
+    console.log(message, ...optionalParams);
+  }
+}
+
+async function main() {
+  const client = new Client({
+    user: DB_USERNAME,
+    host: DB_HOST,
+    database: DB_NAME,
+    password: DB_PASSWORD,
+    port: Number(DB_PORT),
+  });
+
+  const start = performance.now(); // Start profiling
+
+  try {
+    await client.connect();
+
+    // Fetch and write navbar, footer, and config JSONs
+    await fetchAndWriteSiteData(client);
+
+    // Fetch all resources and their full permalinks
+    const resources = await getAllResourcesWithFullPermalinks(client);
+
+    // Process each resource
+    for (const resource of resources) {
+      logDebug(
+        `Processing resource with id ${resource.id}, fullPermalink: ${resource.fullPermalink}`
+      );
+      if (resource.type === "Page" && resource.content) {
+        await writeContentToFile(
+          resource.fullPermalink,
+          resource.content,
```

Comment:
issue: I think there needs to be some adjustments to include the props that have been moved out of the `content` column (e.g. page title).

Reply:
Added in line 77